blueflood-http/src/main/java/com/rackspacecloud/blueflood/outputs/handlers/HttpMetricDataQueryServer.java

```diff
@@ -61,15 +61,21 @@ public void startServer() {
         RouteMatcher router = new RouteMatcher();
         router.get("/v1.0", new DefaultHandler());
         router.get("/v1.0/:tenantId/experimental/views/metric_data/:metricName", new HttpRollupsQueryHandler());
+
         router.post("/v1.0/:tenantId/experimental/views/metric_data", new HttpMultiRollupsQueryHandler());
+        router.post("/v2.0/:tenantId/views", new HttpMultiRollupsQueryHandler());
 
         router.get("/v2.0", new DefaultHandler());
         router.get("/v2.0/:tenantId/views/:metricName", new HttpRollupsQueryHandler());
-        router.post("/v2.0/:tenantId/views", new HttpMultiRollupsQueryHandler());
         router.get("/v2.0/:tenantId/metrics/search", new HttpMetricsIndexHandler());
         router.get("/v2.0/:tenantId/metric_name/search", new HttpMetricTokensHandler());
         router.get("/v2.0/:tenantId/events/getEvents", new HttpEventsQueryHandler(getEventsIO()));
-        router.options("/v2.0/:tenantId/events/getEvents", new HttpEventsQueryHandler(getEventsIO()));
+
+        router.options("/v2.0/:tenantId/views/:metricName", new HttpOptionsHandler());
+        router.options("/v2.0/:tenantId/views", new HttpOptionsHandler());
+        router.options("/v2.0/:tenantId/metrics/search", new HttpOptionsHandler());
+        router.options("/v2.0/:tenantId/metric_name/search", new HttpOptionsHandler());
+        router.options("/v2.0/:tenantId/events/getEvents", new HttpOptionsHandler());
 
```

Comment:
@shintasmith @ChandraAddala @usnavi Hi guys, note here that I hard coded all these OPTIONS request paths to be individual lines all with the ame HttpOptionsHandler.  However, in theory we can just make this all one line with a regex that supports any OPTIONS requests with the path pattern matching "/v2.0/:tenantId/.*" which should covers any path starting with that pattern, and return the CORS response headers.  

Do you guys prefer it this way (with the very specific paths that allows for OPTIONS request method)?  Or would you prefer opening it to any paths with some sort of common regex pattern?